orcid-core/src/main/java/org/orcid/core/manager/read_only/impl/AffiliationsManagerReadOnlyImpl.java

```diff
@@ -124,8 +123,7 @@ public EmploymentSummary getEmploymentSummary(String userOrcid, Long employmentI
      * @return the list of employments that belongs to this user
      * */
     @Override
-    @Cacheable(value = "employments-summaries", key = "#userOrcid.concat('-').concat(#lastModified)")
-    public List<EmploymentSummary> getEmploymentSummaryList(String userOrcid, long lastModified) {
+    public List<EmploymentSummary> getEmploymentSummaryList(String userOrcid) {
         List<OrgAffiliationRelationEntity> employmentEntities = orgAffiliationRelationDao.getByUserAndType(userOrcid, AffiliationType.EMPLOYMENT);
```

Comment:
The managers on 2.0 and 3.0_dev1 are not using the same dao methods, for example: 
The 2.0 AffiliationsManagerReadOnlyImpl.getEmploymentSummaryList uses orgAffiliationRelationDao.**getByUserAndType** while the 3.0 AffiliationsManagerReadOnlyImpl uses orgAffiliationRelationDao.**getEducationSummaries**, I think we should make them consistent, please iterate over the managers and verify that, for the same task, they are calling the same dao objects (unless there is a valid reason not to do it)